src/SourceBuild/content/eng/build.targets

```diff
@@ -0,0 +1,29 @@
+<Project>
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetSingleArchiveItem" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion" >
+
+      <GetSingleArchiveItem SdkArchiveItems="@(SdkTarballItem)">
```

Comment:
Instead of a selecting a single item, would it be better to just compare each possible item?

Reply:
And error if the number of possible comparisons is zero

Reply:
I only needed this because Windows has a .zip, and .msi and .msi.zip in SdkTarballItem, and I needed to choose the .zip. Is there a scenario where there would be multiple valid archives that it would make sense to check multiple?

Reply:
Ahh I see. I would check the .zip in that case, not the wixpack.zip, and explicitly exclude wixpacks from that input set. Other repos may have multiple archives though. Runtime has the main runtime zip, then a few others (targeting pack?)

Reply:
I changed that task to GetValidArchiveItems that returns all valid items and errors if there are none, then assert afterwards that there's exactly one valid archive.